pyvista/examples/examples.py

```diff
@@ -412,3 +412,65 @@ def load_nut():
 
     """
     return pyvista.read(os.path.join(dir_path, 'nut.ply'))
+
+
+def _cart2sphe(x, y, z):
```

Comment:
I was about to point out that we have
https://github.com/pyvista/pyvista/blob/c0d5fc7c498504261aaf3e56c90f8c001ae65d67/pyvista/utilities/features.py#L137-L138
but apparently that function doesn't do what I'd expect it to do...

Reply:
Moving this to utilities where it should be...